.travis.yml

```diff
@@ -143,6 +145,13 @@ matrix:
         - QUERY_MODE=extendedCacheEverything
         - COVERAGE=N
         - TZ=Europe/Moscow # +03:00, no DST
+    - jdk: oraclejdk8
+      sudo: required
+      addons:
+        postgresql: "9.6"
+      env:
+        - PG_VERSION=9.6
+        - SSLTEST=Y
```

Comment:
Does it make sense to incorporate SSLTEST into existing `xa / replication / ...` job just to keep Travis execution "fast"?

Reply:
I've had to muck with pg_hba.conf to get this to work. If we can get both to work then fine